test/scaleTest.js

```diff
@@ -0,0 +1,52 @@
+/* global suite test assert */
+
+function checkTransformKeyframes (keyframes) {
+  var target = document.createElement('div');
+
+  for (var value of keyframes) {
+    target.style.transform = '';
+    target.style.transform = value;
+    assert.notEqual(target.style.transform, '',
+      'Invalid expected keyframe: ' + value);
+  }
+}
+
+function isAnimationEqual (actualKeyframes, expectedKeyframes) {
+  var timing = {duration: 1};
+  var currentTimes = [0.10, 0.25, 0.50, 0.75, 0.90];
+
+  checkTransformKeyframes(expectedKeyframes.transform);
+
+  for (var i = 0; i < currentTimes.length; i++) {
+    var currentTime = currentTimes[i];
+
+    var actualTarget = document.createElement('div');
+    document.body.appendChild(actualTarget);
+
+    var expectedTarget = document.createElement('div');
+    document.body.appendChild(expectedTarget);
+
+    var animation = actualTarget.animate(actualKeyframes, timing);
+    animation.currentTime = currentTime;
+    var result = window.getComputedStyle(actualTarget).transform;
+
+    animation = expectedTarget.animate(expectedKeyframes, timing);
+    animation.currentTime = currentTime;
+    var expected = window.getComputedStyle(expectedTarget).transform;
+
+    actualTarget.parentNode.removeChild(actualTarget);
+    expectedTarget.parentNode.removeChild(expectedTarget);
+
+    assert.equal(result, expected, 'test failed for scale: ' + actualKeyframes.scale + ' transform: ' + expectedKeyframes.transform);
```

Comment:
The message can just be `'at currentTime ' + currentTime + ' using ' + actualKeyframes + ' ' + expectedKeyframes` since the keyframes already say scale/transform. Also this will generalise the helper to work with rotate and translate later.